lib/schema_plus/views/active_record/connection_adapters/postgresql_adapter.rb

```diff
@@ -2,17 +2,12 @@ module SchemaPlus::Views
   module ActiveRecord
     module ConnectionAdapters
       module PostgresqlAdapter
-
-        def views(name = nil) #:nodoc:
-          SchemaMonkey::Middleware::Schema::Views.start(connection: self, query_name: name, views: []) { |env|
-            sql = <<-SQL
-            SELECT viewname
-              FROM pg_views
-            WHERE schemaname = ANY (current_schemas(false))
-            AND viewname NOT LIKE 'pg\_%'
-            SQL
-            sql += " AND schemaname != 'postgis'" if adapter_name == 'PostGIS'
```

Comment:
What's the deal with comparing the schemaname and ignoring PostGIS?
I'm not really familiar with PostGIS, but I guess it does something strange here?
@ronen do you have any suggestions what to do with that?